scancode/scanCode.py

```diff
@@ -549,7 +554,7 @@ def is_dir(path):
             print_warning(WARN_SCAN_EXCLUDED_PATH % excluded_path)
 
     # Display which files where excluded from these paths
-    if args.display_exclusions:
+    if VERBOSE:
```

Comment:
Sometimes the listing of exclusions is not desirable unto itself (even with VERBOSE) enabled; would prefer leaving this as a sep. option in addition to general verbose.

Reply:
Reverted.